crypto/src/merkle/mod.rs

```diff
@@ -0,0 +1,39 @@
+pub mod merkle_path_set;
+pub mod merkle_tree;
+
+use winterfell::crypto::Hasher as HashFn;
+
+pub use winterfell::crypto::hashers::Rp64_256 as Hasher;
+pub use winterfell::math::{
+    fields::{f64::BaseElement as Felt, QuadExtension},
+    ExtensionOf, FieldElement, StarkField,
+};
```

Comment:
We can probably import this from `crate` (since most of them are already imported in `lib.rs`).